dashboard/settings.py

```diff
@@ -45,6 +45,11 @@
 LOGOUT_URL = '/accounts/logout'
 HELP_URL = ENV.get("HELP_URL", "https://sites.google.com/umich.edu/my-learning-analytics-help/home")
 
+URL_VIEW_RESOURCES_ACCESSED =       ENV.get("URL_VIEW_RESOURCES_ACCESSED", "https://its.umich.edu/academics-research/teaching-learning/my-learning-analytics/support/resources-accessed")
+URL_VIEW_ASSIGNMENT_PLANNING_V1 =   ENV.get("URL_VIEW_ASSIGNMENT_PLANNING_V1", "https://its.umich.edu/academics-research/teaching-learning/my-learning-analytics/support/assignment-planning")
+URL_VIEW_ASSIGNMENT_PLANNING =   ENV.get("URL_VIEW_ASSIGNMENT_PLANNING", None)
+URL_VIEW_GRADE_DISTRIBUTION =       ENV.get("URL_VIEW_GRADE_DISTRIBUTION", "https://its.umich.edu/academics-research/teaching-learning/my-learning-analytics/support/grade-distribution")
```

Comment:
@chrisrrowland, it'd be nice to build these URLs off of a base, maybe `INFO_URL`. I'm assuming that the `HELP_URL` needs to be different. If they are the same (@jennlove-um), then we should try to consolidate, and we'd need to think about updating the `AvatarModal` code to use what you're doing here. 

Also, I'd delete the spaces after the assignment operators (e.g. `URL_VIEW_RESOURCES_ACCESSED = ENV...`)

Reply:
I removed the extra indentation.

However I like the flexibility of having the full url rather than sharing a prefix.  Odds are a common prefix would work, but this way allows admins to point at their documentation no matter where it is, rather than requiring it to all be co-located.

Reply:
I think having documentation that isn't co-located might be a problem, but certainly this does provide more flexibility. Not a huge deal. I'm more concerned about whether the `HELP_URL` default and `AvatarModal` components needs to be updated.

Reply:
I have updated those per @pushyamig to open in a new tab, is [this](https://github.com/tl-its-umich-edu/my-learning-analytics/pull/1008/commits/7d8997d8f73dcf5e96541535e63e96a01bc9d362) what you meant?

Reply:
No, this is a separate problem. Essentially, you're improving on (but also replicating) a different help URL set up that I referred you to when you started working on this issue: https://github.com/tl-its-umich-edu/my-learning-analytics/blob/f9b613a96a2ec4c84870eb2a920fce8d416a7c85/assets/src/components/AvatarModal.js#L96-L105

I checked with @jennlove-um, and the links from the `AvatarModal` and the `SelectCard`s should be the same. Now, you could configure them to be the same, but I really think these should be populated in the same way. Am I making more sense now?

Reply:
Yes, it should be a target blank since it is external link. Thanks for doing the @chrisrrowland

Reply:
You can keep the `HELP_URL` settings variable (in fact you have to, since there should be a help landing page linked to somewhere), but I'd rather see this logic shown above replaced with some way of pulling these other globals you've set up in MyLA globals.

Reply:
Hmm, well i see why you wanted a base url in that case.   I did not understand that these urls were supposed to be related in any way to those urls.  Is that the case?  Are those URLs supposed to be replaced by the new ones?  That was definitely not clear to be based on the issue description, but maybe it assumed I had more insight into the issue than I did.

@jennlove-um how is this supposed to work?  Are the info links on the select cards supposed to link to the same context aware help links for each of the views, or are these independent links?

Reply:
The info links on the select cards should be context aware like each visualization is.  When an instructor wants to learn more about one of the views, it makes sense to take them directly to the help page for that view.

Reply:
@ssciolla I refactored to be more consistent with how help urls are used